pkg/integration/integration.go

```diff
@@ -3,12 +3,23 @@ package integration
 import (
 	"fmt"
 	"os"
+	"strings"
 
 	color "github.com/logrusorgru/aurora"
 )
 
 func Print() {
-	fmt.Println(bashSource)
+	var currentShell = os.Getenv("SHELL")
+
+	fmt.Println(shellSource)
```

Comment:
Should we print most of the source already if we have a major failure condition right after?

If we are running an unsupported shell, I think we should not print anything to stdout, because it will be eval'ed... by this unsupported shell.

I know I mentioned the general idea of preferring warnings rather than stopping execution...(because it could partially work and that could be enough for some people) but in this case, it looks like it *could* actually be very dangerous...

Reply:
Yeah you're right it should work without shell integration but we should not have a half one